packages/salesforcedx-vscode-core/src/commands/isvdebugging/bootstrapCmd.ts

```diff
@@ -192,6 +256,36 @@ export class IsvDebugBootstrapExecutor extends SfdxCommandletExecutor<{}> {
       cancellationToken
     );
 
+    // 2b: update sfdx-project.json with namespace
+    const orgNamespaceInfoResponseJson = await this.executeCommand(
+      this.buildQueryForOrgNamespacePrefixCommand(response.data),
+      { cwd: projectPath },
+      cancellationTokenSource,
+      cancellationToken
+    );
+    try {
+      const sfdxProjectJsonFile = path.join(projectPath, 'sfdx-project.json');
+      const sfdxProjectConfig = JSON.parse(
+        fs.readFileSync(sfdxProjectJsonFile).toString()
```

Comment:
Can this be given the encoding to get the string form straight from `readFileSync`?

Reply:
Might be a good idea. I'll check the api and enforce UTF-8.

Reply:
@DatGuyJonathan I applied encoding to all file ops (read and write). Good catch. Even though we shouldn't have unexpected chars in those files it's probably best to have it for Windows.